projects/bbb_canbus_listener/CMakeLists.txt

```diff
@@ -11,4 +11,4 @@ set(${PROJECT_NAME}_SRCS main.cpp)
 add_executable(${PROJECT_NAME} ${${PROJECT_NAME}_SRCS})
 target_link_libraries(${PROJECT_NAME} ${ZMQ_LIBRARIES} ${PROTOBUF_LIBRARIES} nt_client)
 
-include_directories(uccm-sensors)
+include_directories(../../src/uccm-sensors)
```

Comment:
Can you move the include_directories into the src/cmakelists.txt andremove this one